docs/README.md

```diff
@@ -664,6 +665,72 @@ If you followed [Scraping metrics using the OpenShift monitoring stack](#scrapin
             enabled: true
           url: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091
     ```
+#### Integrating Kiali with OpenShift Distributed Tracing
+This section describes how to setup Istio and Kiali with OpenShift Distributed Tracing to send and receive distributed traces.
+
+*Prerequisites*
+* A Tempo stack is installed and configured
+* An instance of an OpenTelemetry collector is already configured in the istio-system namespace
+* An Istio instance is created with the OpenShift profile
+* An Istio CNI instance is created with the OpenShift profile
+
+*Steps*
+1. Setup Istio to include the OpenTelemtry settings:
+    ```yaml
+    meshConfig:
+      enableTracing: true
+      extensionProviders:
+      - name: otel-tracing
+        opentelemetry:
+          port: 4317
+          service: otel-collector.istio-system.svc.cluster.local 
+    ```
+service: The service of the OpenTelemetry collector in the istio-system namespace.
+
+2. Create an Istio telemetry resource to active the OpenTelemetry tracer
+    ```yaml
+    apiVersion: telemetry.istio.io/v1
+    kind: Telemetry
+    metadata:
+    name: otel-demo
+    namespace: istio-system
+    spec:
+    tracing:
+    - providers:
+        - name: otel-tracing
+          randomSamplingPercentage: 100
+    ```
+3. Setup Kiali to access traces in the Tempo frontend: 
+    ```yaml
+    external_services:
+      tracing:
+        enabled: true
+        provider: tempo
+        use_grpc: false
+        in_cluster_url: http://tempo-sample-query-frontend.tempo:3200
+        url: 'https://tempo-sample-query-frontend-tempo.apps-crc.testing'
+    ```
+
+4. To see the "View in Tracing" link, Grafana should be enabled:
```

Comment:
```suggestion
4. Enable Grafana to see the "View in Tracing" link:
```

Reply:
Is this editing the same Kiali CR as above? I know one of these steps is "setup tracing" whereas the other is "setup grafana" but would it make sense to combine them so that the user doesn't have to edit the same yaml twice?